src/libraries/System.Net.Http/tests/StressTests/HttpStress/run-docker-compose.ps1

```diff
@@ -73,6 +73,16 @@ finally {
 
 if (!$buildOnly)
 {
+    $env:IMAGE_SHARE="C:/share"
```

Comment:
Why are there 2 places IMAGE_SHARE is set? in run-docker-compose.ps1/sh and in http.yml. Is there one source of truth? Will it still work if at some point the values will diverge?
Also, should IMAGE_SHARE be checked on being already set the way CLIENT_SHARE and SERVER_SHARE are checked?

Reply:
`docker-compose up` is either called via Azure Pipeline from http.yml, or if you run it locally via `run-docker-compose.ps1`, so it have to be set from 2 places. I really didn't want to change the way the pipeline/local runs are done in this PR, so I added it to both places.

>  should IMAGE_SHARE be checked

No, it's basically constant you don't want anyone to override.